File: crates/laya-parse/tests/fixtures/refs_sample.py
import os.path
from typing import Optional
from .store import MoonStore, open_store as opener


class Cache(BaseCache):
    def get(self, key: str) -> Optional[Entry]:
        store = MoonStore(key)
        value = store.fetch_value(key)
        print(value)
        return helper(value)


def helper(v):
    return len(transform(v))

File: crates/laya-parse/tests/fixtures/RefsSample.java
package com.acme.cache;

import java.util.List;
import com.acme.store.MoonStore;

public class Cache extends BaseCache {
    private final MoonStore store = new MoonStore();

    public Entry lookup(String key) {
        List<Entry> hits = store.fetchAll(key);
        System.out.println(hits);
        return decode(hits.get(0));
    }
}
